src/app/tab/tab.component.html

```diff
@@ -32,6 +32,14 @@
           </tab-time>
         </ng-container>
       </ng-container>
+      <ng-container *ngIf="settings.config.covidData.enabled && settings.config.covidData.countries.length > 0">
+        <ng-container *ngFor="let c of settings.config.covidData.countries; let i = index;" >
+          <tab-covid *ngIf="c.position === 'nw'" [index]="i" [countryCode]="c.code"
+          [style.order]="shared.getOrder(c.id)"
```

Comment:
Make sure to pass in the elements position as a parameter in the getOrder function.